### PR TITLE
Quickstart Changes

### PR DESCRIPTION
**Pull request by calldelegation**
- [x] Please add proper labels
- [x] If there is an issue associated with this PR, please link the issue (right-hand sidebar)
- [x] If there is not an issue associated with this PR, add this PR to the "Fuel Indexer" project (right-hand sidebar)

### Description

Fixes a lot of issues that grantees were complaining about in the previous indexer quickstart. This version is not compatible with beta-3 but the latest version of forc which has fuel-indexer at `0.18.0`

### Testing steps

run `npx markdownlint docs/src/getting-started/quickstart.md` in project root

### CODE REVIEW

**Comment by calldelegation** [comment]
Sorry for the pings still in draft

File: docs/src/getting-started/quickstart.md [hunk 1]
```diff
@@ -6,91 +7,141 @@ In this tutorial you will:
...
+<!-- markdownlint-restore -->
 
-> If you require a non-default `fuelup` installation, please [read the `fuelup` installation docs.](https://github.com/FuelLabs/fuelup)
+3. [Install the Fuel toolchain](https://github.com/FuelLabs/fuelup).
 
-### 1.2 WebAssembly (WASM) Setup
+Make sure you have the latest version of `fuelup` by running the following command:
 
-Indexers are typically compiled to WASM so you'll need to have the proper WASM compilation target available on your system. You can install this target using `rustup`:
+Note: older versions of `fuel-indexer` might not work in this guide
```
Comment (deekerno) [inline docs/src/getting-started/quickstart.md:30]: Consider moving this line under the `fuelup self update` line and changing to:
> Check that you have the latest available version of `fuel-indexer` by ensuring that your toolchain is up-to-date.

It's a bit more helpful and doesn't make an assumption of the toolchain that they're using.

**Review by deekerno — CHANGES_REQUESTED** [review]

File: docs/src/getting-started/quickstart.md [hunk 2]
```diff
@@ -6,91 +7,141 @@ In this tutorial you will:
 2. Create, build, and deploy an indexer to an indexer service hooked up to Fuel's `beta-3` testnet.
 3. Query your newly created index for data using GraphQL.
 
-## 1. Setting up your environment
+## 1. Prerequisites
 
-In this Quickstart, we'll use Fuel's toolchain manager [`fuelup`](https://github.com/FuelLabs/fuelup) in order to install the `forc-index` component that we'll use to develop our indexer.
+In this Quickstart, we'll use Fuel's toolchain manager [`fuelup`](https://github.com/FuelLabs/fuelup) in order to install the `forc-index` component that we'll use to develop our indexer. There are a lot of prerequisites so please read through each installation carefully.
```
[the remark this answers is not in the document]
Comment (ra0x3) [reply docs/src/getting-started/quickstart.md:12]: @calldelegation defer'ing to @deekerno here but yes, short and sweet 😄

File: docs/src/getting-started/quickstart.md [hunk 3]
```diff
@@ -6,91 +7,141 @@ In this tutorial you will:
...
+
+Save and close the file by pressing `Ctrl+O` followed by `Ctrl+X`
+
+Apply the changes without opening a new terminal by running:
+
+<!-- markdownlint-disable MD014 -->
+```console
+$ source ~/.zshrc
 ```
+<!-- markdownlint-restore -->
```
[the remark this answers is not in the document]
Comment (calldelegation) [reply docs/src/getting-started/quickstart.md:94]: Nope i agree with you here

**Review by deekerno — COMMENTED** [review]
I didn't realize this on the first review pass, but we're asking users to install Postgres but then we instruct them to use the embedded database feature. We should choose one and stick with it.

@ra0x3: What do you think?

**Comment by calldelegation** [comment]
> I didn't realize this on the first review pass, but we're asking users to install Postgres but then we instruct them to use the embedded database feature. We should choose one and stick with it.
> 
> @ra0x3: What do you think?

From the conversation on slack

**Comment by ra0x3** [comment]
@calldelegation Following up here. Is this PR still being actively worked on?

**Comment by deekerno** [comment]
Closing this for now. We'll be revamping a lot of our documentation as part of #1174, so we'll move any relevant changes into that.

Thanks! ❤️

**Pull request closed without merging**